haas/ext/switches/_console.py

```diff
@@ -119,6 +119,16 @@ def revert_port(self, port):
         self.exit_if_prompt()
         self.console.expect(self.config_prompt)
 
+    def _save_check(self, switch_type):
```

Comment:
How about a name that makes the function's job a little more clear, e.g. `_should_save`? `_save_check` could be a number of other things, c.f. the check_call, check_output functions from the subprocess module.